extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLUIConfig.java

```diff
@@ -17,12 +19,7 @@
      * Always include the UI. By default this will only be included in dev and test.
      * Setting this to true will also include the UI in Prod
      */
-    @ConfigItem(defaultValue = "false")
-    boolean alwaysInclude;
+    @ConfigItem
+    Optional<Boolean> alwaysInclude;
```

Comment:
I don't understand this change? We certainly prefer having proper default values.

Reply:
That is the only way I could get it to work like it did before. The proper default value now is empty (not explicitly set by the user). That way I can still so the "include in dev and test" by default and "exclude" in prod by default. If the user set this explicitly, then I do not include at all everywhere (if false) or also include in prod (if true). Else there is no way to exclude in dev and test (i.e don't use this at all)